DariaCagle3(optional).cpp

```diff
@@ -0,0 +1,125 @@
+#include <ctime>
+#include <iostream>
+#include <random>
+
+enum TASKS { DELETE = 1, UPPER, DIGIT, LOWER };
+
+constexpr int kCodeDif{32};
+
+void deleteItem() {
+  constexpr int length{5};
+  int array[length]{};
+  constexpr int min{1};
+  constexpr int max{20};
+  std::mt19937 gen(time(0));
+  std::uniform_int_distribution<> num(min, max);
+  for (int i{0}; i < length; i++) {
+    array[i] = num(gen);
+  }
```

Comment:
или использоать range for
```suggestion
  for (auto& value : array) {
    value = num(gen);
  }
```

Reply:
согласна. поменяла